### PR TITLE
cleaning SWA

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

cleaning up potentially misleading condition
using full file name

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by carmocca** [comment]
Maybe it's fine to not deprecate since it's in beta and most people will import it as `from pytorch_lightning.callbacks import StochasticWeightAveraging`

**Comment by Borda** [comment]
> Maybe it's fine to not deprecate since it's in beta and most people will import it as `from pytorch_lightning.callbacks import StochasticWeightAveraging`

good point, done

File: pytorch_lightning/callbacks/stochastic_weight_avg.py [hunk 1]
```diff
@@ -102,12 +102,10 @@ def __init__(
...
-        if (
-            swa_lrs is not None and (
-                not isinstance(swa_lrs, (float, list)) or isinstance(swa_lrs, float) and swa_lrs <= 0
-                or isinstance(swa_lrs, list) and not all(lr > 0 and isinstance(lr, float) for lr in swa_lrs)
-            )
-        ):
+        _wrong_type = not isinstance(swa_lrs, (float, list))
+        _wrong_float = isinstance(swa_lrs, float) and swa_lrs <= 0
+        _wrong_list = isinstance(swa_lrs, list) and not all(lr > 0 and isinstance(lr, float) for lr in swa_lrs)
+        if (swa_lrs is not None and (_wrong_type or _wrong_float or _wrong_list)):
```
Comment (carmocca) [inline pytorch_lightning/callbacks/stochastic_weight_avg.py:108]: No need for underscore

```suggestion
        wrong_type = not isinstance(swa_lrs, (float, list))
        wrong_float = isinstance(swa_lrs, float) and swa_lrs <= 0
        wrong_list = isinstance(swa_lrs, list) and not all(lr > 0 and isinstance(lr, float) for lr in swa_lrs)
        if (swa_lrs is not None and (wrong_type or wrong_float or wrong_list)):
```

**Review by carmocca — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by akihironitta — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Pull request merged**